localstack/config.py

```diff
@@ -230,22 +230,33 @@ def is_env_not_false(env_var_name: str) -> bool:
     return os.environ.get(env_var_name, "").lower().strip() not in FALSE_STRINGS
 
 
-def load_environment(profile: str = None) -> Optional[str]:
-    """Loads the environment variables from ~/.localstack/{profile}.env
-    :param profile: the profile to load (defaults to "default")
-    :returns str: the name of the actually loaded profile (might be the fallback)
+def load_environment(profiles: str = None, env=os.environ) -> List[str]:
+    """Loads the environment variables from ~/.localstack/{profile}.env, for each profile listed in the profiles.
+    :param env: environment to load profile to. Defaults to `os.environ`
+    :param profiles: a comma separated list of profiles to load (defaults to "default")
+    :returns str: the list of the actually loaded profiles (might be the fallback)
     """
-    if not profile:
-        profile = "default"
-
-    path = os.path.join(CONFIG_DIR, f"{profile}.env")
-    if not os.path.exists(path):
-        return None
+    if not profiles:
+        profiles = "default"
 
+    profiles = profiles.split(",")
+    environment = {}
     import dotenv
 
-    dotenv.load_dotenv(path, override=False)
-    return profile
+    for profile in profiles:
+        profile = profile.strip()
+        path = os.path.join(CONFIG_DIR, f"{profile}.env")
+        if not os.path.exists(path):
+            print(f"Profile '{profile}' specified, but file {path} not found.")
+            continue
+        environment |= dotenv.dotenv_values(path)
```

Comment:
```suggestion
        environment.update(dotenv.dotenv_values(path))
```

much easier to read :-)